code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -7453,6 +7453,44 @@ var/global/list/bomb_like_items = list(/obj/item/device/transfer_valve, /obj/ite
 	reagents.add_reagent(ZAMSPICYTOXIN, 5)
 	bitesize = 3
 
+/obj/item/weapon/reagent_containers/food/snacks/blethernoodlesoup
+	name = "Blether Noodle Soup"
+	desc = "A hearty grey noodle soup. Great for teaching growing greylings new words! Not to be confused with human alphabet soup."
+	icon_state = "blethernoodlesoup_open"
+	trash = /obj/item/trash/soupcantrash
+	food_flags = FOOD_MEAT | FOOD_LIQUID
+	crumb_icon = "dribbles"
+	filling_color = "#FF9700"
+	valid_utensils = UTENSILE_SPOON
```

Comment:
Bro you tellin me you eat noodles WITH A SPOON?

Reply:
Maybe

Reply:
I made it so you can also use a fork

Reply:
based.
Prepare for the (un)forseen consequence of the fork having a tiny blether soup can because that's how the forks are coded. (keep it that way, it's dumb and I like it)